manager/apis/app/v1alpha1/utils/configuration.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"os"
+)
+
+// GetDataDir returns the directory where the data resides.
+func GetDataDir() string {
+	return os.Getenv("DATA")
```

Comment:
should we add here some default, e.g `/tmp` ?

Reply:
nor sure about that as if the manager runs inside Kubernetes the default value is /data while locally its /tmp.

Reply:
OK, where this file should reside? does `pkg/environment/` the right place? Thanks

Reply:
Yes, as long as we don't have conflicts between manager env and env of other components.

Reply:
I moved [manager/controllers/utils/configuration.go → pkg/environment/environment.go](https://github.com/fybrik/fybrik/pull/1512/files#diff-616422b9b983db329c853b3e73274dfdb95b49add7e27bcfd47257923eead1e8)